### PR TITLE
updating how wsgi and nginx interact

### PR DESCRIPTION
**Pull request by orangeblossomest**

### CODE REVIEW

File: infrastructure/docker-compose.yml [hunk 1]
```diff
@@ -6,15 +6,14 @@ services:
...
-      - "7999:8080"
       - "8000:8000"
+      - "40:40"
       - "80:80"
-      - "30:30"
     expose:
       - "8000"
-      - "443"
       - "80"
       - "30"
```
Comment (Choski) [inline infrastructure/docker-compose.yml:15]: Can we remove port 30 as it should only be exposed on webportal right..... waf(40) to webportal(30)

File: web_application_firewall/Dockerfile [hunk 2]
```diff
@@ -4,6 +4,8 @@ WORKDIR /
 
 COPY . .
 
+HEALTHCHECK NONE
+
 EXPOSE 8000
```
Comment (Choski) [inline web_application_firewall/Dockerfile:9]: Theres duplication of exposes here

**Review by Choski — COMMENTED** [review]
Added two comments for general cleanup

File: infrastructure/docker-compose.yml (hunk 1 above)
Comment (orangeblossomest) [reply infrastructure/docker-compose.yml:15]: ah yes i forgot to do that sorry let me do so now - had re-addded during testing

**Pull request merged**